app/static/css/main.css

```
@@ -11,4 +11,14 @@
 }
 .card-header > a[aria-expanded="true"]:after {
   content: "\f077"; /* fa-chevron-up */
-}
\ No newline at end of file
+}
+.navbar-collapse {
+  align-items: flex-end;

```

```suggestion
  align-items: flex-end;
  flex-grow: unset;
}
```

by adding the `navbar-collapse` class this shifts the navbar to the left, for now let's keep it all right aligned as that is where the collapse button will appear and all items will be aligned within there.